UNITY.md

```diff
@@ -0,0 +1,160 @@
+# CAVE Unity Project
```

Comment:
Can you add in some setup instructions? Seems like most of the stuff in the README is just want exists in this project which is good too!

Reply:
Or is the README already the set up instructions? It seems more like an FYI than instructions

Reply:
Oohhh oops. This is `UNITY.md` not the readme for the whole repo! All good